Unity 2023: Ambiguous method

`Library\PackageCache\com.realitycollective.utilities@1.0.5-pre.2\Runtime\Utilities\WebRequestRest\Rest.cs(822,17): error CS0121: The call is ambiguous between the following methods or properties: 'RealityCollective.Utilities.Async.AwaiterExtensions.GetAwaiter(UnityEngine.AsyncOperation)' and 'UnityEngine.AsyncOperationAwaitableExtensions.GetAwaiter(UnityEngine.AsyncOperation)'
`

--- a/Runtime/Utilities/Async/AwaiterExtensions.cs
+++ b/Runtime/Utilities/Async/AwaiterExtensions.cs
@@ -87,8 +87,12 @@ namespace RealityCollective.Utilities.Async
         public static SimpleCoroutineAwaiter GetAwaiter(this WaitWhile instruction)
             => GetAwaiterReturnVoid(instruction);
 
+#if !UNITY_2023_1_OR_NEWER
+
         public static SimpleCoroutineAwaiter<AsyncOperation> GetAwaiter(this AsyncOperation instruction)
             => GetAwaiterReturnSelf(instruction);
+
+#endif
 
         public static SimpleCoroutineAwaiter<Object> GetAwaiter(this ResourceRequest instruction)
         {

